Recipes/Recipes.API/Program.cs

```diff
@@ -13,40 +12,23 @@
 
 var app = builder.Build();
 
+using var scope = app.Services.CreateScope();
```

Comment:
This scope essentially lasts the whole lifetime of the application, it would be wise to dispose of it sooner (after you're done making db migrations)